.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/github.com/go-swagger/go-swagger"]
+	path = src/github.com/go-swagger/go-swagger
+	url = git@github.com:go-swagger/go-swagger
```

Comment:
Actually, please use `https://github.com/…` for public repositories. It makes it easier to deal with from CI.

Sorry I didn't catch this on the first pass.

Reply:
Addressed with commit d7195ab4405643e127651d4280d18efb374dcb29